hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieMetadataTableValidator.java

```diff
@@ -197,6 +200,27 @@ public HoodieMetadataTableValidator(JavaSparkContext jsc, Config cfg) {
     this.taskLabels = generateValidationTaskLabels();
   }
 
+  /**
+   * Returns list of Throwable which were encountered during validation. This method is useful
+   * when ignoreFailed parameter is set to true.
```

Comment:
You may also want to note you can use this option to find all issues instead of just the first one

Reply:
Sorry! Did not get this.

Reply:
If i'm not wrong, I think it's about making javadoc more clear that the list of throwables is actually the list of all validation failures (MDT, RLI, etc).

Reply:
Yes, that was the idea. It's used to get back all the things that may be wrong with the MDT